src/commands/commerce/extension/map.ts

```diff
@@ -147,4 +147,33 @@ export class MapExtension extends SfdxCommand {
             return returnResult;
         }
     }
+    private deleteduplicateMaps(extensionName: string, userName: string): void {
+        try {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+            const EPNQuery = forceDataSoql(
+                `SELECT ExtensionPointName FROM RegisteredExternalService WHERE DeveloperName='${extensionName}'`,
+                userName
+            ).result.records[0]['ExtensionPointName'];
+            const existingIds = forceDataSoql(
+                // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+                `SELECT DeveloperName,ExternalServiceProviderType FROM RegisteredExternalService WHERE ExtensionPointName='${EPNQuery}' AND ExternalServiceProviderType='Extension'`,
```

Comment:
this method should also take a webstoreId and an additional where condition for `StoreId='${storeId}'`

Currently it'll delete mapped extensions in other stores